src/core/issues-notes.js

```diff
@@ -244,6 +249,24 @@ function createLabel(label) {
     href="${href}">${name}</a>`;
 }
 
+function htmlToElement(html) {
```

Comment:
Oh. We won't need to do this. We're using [hyperHTML](https://github.com/WebReflection/hyperHTML) in ReSpec. Have a look at other modules (like [core/examples](https://github.com/w3c/respec/blob/develop/src/core/examples.js)) to see how we are using it.

Reply:
Thanks! This was interesting.